Source/Core/Core/DSP/DSPAnalyzer.h

```diff
@@ -20,22 +20,22 @@ namespace DSP
 class Analyzer
 {
 public:
-  explicit Analyzer(const SDSP& dsp);
+  explicit Analyzer();
   ~Analyzer();
 
   Analyzer(const Analyzer&) = default;
```

Comment:
Since none of the {copy,move} {constructors,assignment operators} do anything special anymore, should we get rid of them entirely? I feel like defaulting them explicitly doesn't add much when the compiler already does the same thing with less code.

Reply:
These operators won't actually get generated by the compiler if a user-provided destructor is present. (see the conditions under "Implicitly-declared move constructor" [here](https://en.cppreference.com/w/cpp/language/move_constructor#Implicitly-declared_move_constructor). This also applies to the copy equivalents as well.